README.md

```diff
@@ -1,24 +1,37 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
Going forward, this is the only acceptable [README Template.](https://github.com/microverseinc/readme-template) Take note and use it in your future projects.